src/migrations/1671691725186-User.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey
+} from 'typeorm';
+
+export class User1671691725186 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'users',
+        columns: [
+          {
+            name: 'id',
+            type: 'int',
+            isPrimary: true,
+            isGenerated: true,
+            generationStrategy: 'increment'
+          },
+          {
+            name: 'email',
+            isUnique: true,
+            type: 'varchar',
+            length: '255'
+          },
+          {
+            name: 'cmnd',
+            isUnique: true,
+            type: 'varchar',
+            length: '12'
+          },
+          {
+            name: 'name',
+            type: 'varchar',
+            length: '255'
+          },
+          {
+            name: 'password',
+            type: 'varchar',
+            length: '255'
+          },
+          {
+            name: 'birthday',
+            type: 'date'
+          },
+          {
+            name: 'gender',
+            type: 'varchar',
+            length: '255'
+          },
+          {
+            name: 'wardId',
+            type: 'int'
+          },
```

Comment:
DB convention là column_name
```suggestion
            name: 'ward_id',
            type: 'int'
          },
```